rest/route/service.go

```diff
@@ -127,6 +127,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/admin/service_flags").Version(2).Post().Wrap(requireUser, adminSettings).RouteHandler(makeSetServiceFlagsRouteManager())
 	app.AddRoute("/admin/settings").Version(2).Get().Wrap(requireUser, adminSettings).RouteHandler(makeFetchAdminSettings())
 	app.AddRoute("/admin/settings").Version(2).Post().Wrap(requireUser, adminSettings).RouteHandler(makeSetAdminSettings())
+	app.AddRoute("/admin/task_limits").Version(2).Get().Wrap(requireUser).RouteHandler(makeFetchTaskLimits())
```

Comment:
Everything else under the /admin/ umbrella is (rightly I think) locked down to admins. Would it make sense to make a new umbrella like /global/task_limits or something, to demonstrate that this is purposefully okay to be exposing to outside users, rather than being an admin function?